db/system_keyspace.cc

```diff
@@ -2648,6 +2663,35 @@ future<std::optional<mutation>> system_keyspace::get_group0_schema_version() {
     co_return std::nullopt;
 }
 
+future<std::optional<mutation>> system_keyspace::get_group0_schema_version() {
+    return get_scylla_local_mutation(_db, "group0_schema_version");
+}
+
+static constexpr auto SERVICE_LEVELS_STATUS_KEY = "service_level_v2_status";
+
+future<std::optional<mutation>> system_keyspace::get_service_levels_migration_status_mutation() {
```

Comment:
I wonder why you didn't use integer version? Function name would be shorter and it'd be more natural to extend in the future, like introduce version 3